vmware-event-router/internal/config/v1alpha1/auth.go

```diff
@@ -8,14 +8,16 @@ const (
 	BasicAuth AuthMethodType = "basic_auth"
 	// AWSAccessKeyAuth represents the AWS IAM authentication method using access key and secret key
 	AWSAccessKeyAuth AuthMethodType = "aws_access_key"
+	// AWSIAMRoleAuth represents the AWS IAM authentication method using temporary credentials provided by Security Token Service (STS). Intended for use as IAM role with a Kubernetes service account for use case of running under the Amazon EKS.
```

Comment:
nit: please format this line, i.e. configure line-wrap in your IDE

Reply:
Line-wrap adjusted in 26e4558e0d2e864e25d12f04e3c2e8ae62fe351d